build.gradle

```diff
@@ -25,6 +25,8 @@ subprojects {
         implementation libs.guava
         implementation libs.gson
 
+		implementation libs.bundles.tinylog
```

Comment:
wrong indent here?

Reply:
consequence of the addition of a `.editorconfig` suggesting tabs, while a lot of code uses spaces

Reply:
```suggestion
        implementation libs.bundles.tinylog
```